lib/msal-node/src/network/HttpClient.ts

```diff
@@ -176,14 +177,6 @@ const networkRequestViaProxy = <T>(
                     status: statusCode as number,
                 };
 
-                if ((statusCode < 200 || statusCode > 299) &&
```

Comment:
Is it possible to write any unit tests that excercises the code paths in HttpClient?